frontend/openchat-client/src/openchat.ts

```diff
@@ -4995,6 +4997,31 @@ export class OpenChat extends OpenChatAgentWorker {
         });
     }
 
+    submitProposal(governanceCanisterId: string, proposal: CandidateProposal): Promise<boolean> {
+        return this.sendRequest({
+            kind: "submitProposal",
+            governanceCanisterId: Principal.fromText(governanceCanisterId),
+            proposal,
+        }).then((resp) => {
+            if (resp.kind === "success" || resp.kind === "retrying") {
+                return true;
+            }
+
+            let error = resp.kind;
```

Comment:
what's going on here? Is there something wrong with just logging `resp` if it goes wrong?